src/PKSim.Core/Mappers/IndividualToIndividualBuildingBlockMapper.cs

```diff
@@ -128,23 +134,31 @@ public override IndividualBuildingBlock MapFrom(Individual individual)
       private void addCalculationMethodOriginDataToBuildingBlock(IndividualBuildingBlock buildingBlock, CalculationMethod calculationMethod)
       {
          var repInfo = _representationInfoRepository.InfoFor(RepresentationObjectType.CATEGORY, calculationMethod.Category);
-         addOriginDataToBuildingBlock(buildingBlock, repInfo.DisplayName, calculationMethod.Category);
+         addOriginDataToBuildingBlock(buildingBlock, repInfo.DisplayName, calculationMethod.DisplayName);
       }
 
       private void addOriginDataToBuildingBlock(IndividualBuildingBlock buildingBlock, string key, OriginDataParameter parameter)
       {
          if (parameter == null)
             return;
 
-         addOriginDataToBuildingBlock(buildingBlock, keyForOriginDataParameter(key, parameter), $"{parameter.Value} {parameter.Unit}");
+         var displayValue = originDataFormattedForDisplay(parameter);
+         addOriginDataToBuildingBlock(buildingBlock, keyForOriginDataParameter(key, parameter), $"{displayValue} {parameter.Unit}");
+      }
+
+      private string originDataFormattedForDisplay(OriginDataParameter parameter)
+      {
+         var dimension = _dimensionRepository.DimensionForUnit(parameter.Unit);
+         var unit = dimension.FindUnit(parameter.Unit);
+         return _formatter.Format(dimension.BaseUnitValueToUnitValue(unit, parameter.Value)).TrimEnd('0', '.');
```

Comment:
Is this desirable, or is there another way? I want to remove ```xxx.00``` from displayed values.

Reply:
that definitley feels like a hack lol
what does this do? TrimEnd?